docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -124,6 +124,56 @@ A closing curly bracket (`}`) is ignored if not inside an expression/tag.
 
 TIP: It is also possible to use escape sequences `\{` and `\}` to insert delimiters in the text. In fact, an escape sequence is usually only needed for the start delimiter, ie. `\\{foo}` will be rendered as `{foo}` (no evaluation will happen).
 
+=== Removing Standalone Lines From the Template
+
+By default, Qute parser removes standalone lines from the template output.
+A *standalone line* is a line that contains only section tags (e.g. `{#each}` and `{/each}`), parameter declarations (e.g. `{@org.acme.Foo foo}`) and whitespace characters.
+The default behavior can be disabled by setting the property `quarkus.qute.remove-standalone-lines` to `false`.
+In this case, all whitespace characters from a standalone line will be printed to the output.
+
+.Template Example
+[source,html]
+----
+<html>
+  <body>
+     <ul>
+     {#for item in items} <1>
+       <li>{item.name}</li> <2>
+     {/for} <3>
+     </ul>
+   <body>
+</html>
+----
+<1> This is a standalone line and will be removed.
+<2> This is not a standalone line as it contains non-whitespace characters and an expression.
+<3> This is a standalone line and will be removed.
```

Comment:
Here I'm missing an example of a line with whitespace but no section/parameter, to show that they get kept.